### DISCUSSION TITLE
Custom Client for Linux?

### DISCUSSION THREAD

**Discussion by dom6770** [rustdesk/rustdesk#9178] [Q&A] [answered]
Unfortunately the [docs](https://rustdesk.com/docs/en/self-host/client-configuration/#1-custom-client-generator-pro-only) aren't really specific about which platforms are supported. Are Linux builds supported yet? If not, when will it be available?

We are looking to replace AnyDesk, but without custom linux builds, we won't switch.

**Comment by rustdesk** [comment, marked as answer]
https://twitter.com/rustdesk/status/1788905463678951787
